sdc/tests/test_series.py

```diff
@@ -3867,13 +4067,53 @@ def pyfunc(series, periods, freq, axis, fill_value):
         cfunc = self.jit(pyfunc)
         freq = None
         axis = 0
-        for data in test_global_input_data_float64:
-            series = pd.Series(data)
-            for periods in [-2, 0, 3]:
-                for fill_value in [9.1, np.nan, -3.3, None]:
-                    jit_result = cfunc(series, periods, freq, axis, fill_value)
-                    ref_result = pyfunc(series, periods, freq, axis, fill_value)
-                    pd.testing.assert_series_equal(jit_result, ref_result)
+        datas = test_global_input_data_signed_integer64 + test_global_input_data_float64
+        for data in datas:
+            for periods in [1, 2, -1]:
+                for fill_value in [-1, 0, 9.1, np.nan, -3.3, None]:
+                    with self.subTest(data=data, periods=periods, fill_value=fill_value):
+                        series = pd.Series(data)
+                        jit_result = cfunc(series, periods, freq, axis, fill_value)
+                        ref_result = pyfunc(series, periods, freq, axis, fill_value)
+                        pd.testing.assert_series_equal(jit_result, ref_result)
+
+    @sdc_limitation
```

Comment:
Can you please add a docstring here describing what the limitation is?

Reply:
added